src/lib.rs

```diff
@@ -517,6 +534,25 @@ fn pan_orbit_camera(
         }
 
         // 4 - Update the camera's transform based on current values
+        if let Some(radius) = pan_orbit.radius {
```

Comment:
Here, you are unwrapping the radius in `if let`, then `map`ing over it, and finally `unwrap`ing it at the end. We can replace all that with a single `map`, because none of this code makes sense if the radius is `None`.